Sources/FeaturevisorSDK/Instance+Feature.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import FeaturevisorTypes
+
+extension FeaturevisorInstance {
+
+    func findForceFromFeature(
+        _ feature: Feature,
+        context: Context,
+        datafileReader: DatafileReader) -> Force? {
+
+//            if (!feature.force) { // TODO: Convert to swift
```

Comment:
@fahad19 In JS, does it mean that 'force' array is not undefined, yes?

Reply:
@polok: you are correct.

in JS, `!feature.force` will be true if it is:

- `undefined`
- `null`
- `false` or 
- `0`
- the key `force` in `feature` object does not exist

Reply:
PR updated